net-libs/libtelnet/libtelnet-9999.ebuild

```diff
@@ -22,3 +22,8 @@ IUSE=""
 
 # needed unconditionally for man pages
 DEPEND="app-doc/doxygen"
```

Comment:
BDEPEND, no?

Reply:
Yep, I missed that

Reply:
Np! I often find it's way easier to write things right myself rather than improving other peoples' ebuilds. It is a skill and it takes time, but you will get there (you also end up building up a list of things you want to fix usually!)

Reply:
Yeah, I definitely agree ;)